src/main/java/com/powsybl/openloadflow/ac/PhaseControlOuterLoop.java

```diff
@@ -43,7 +43,7 @@ public void initialize(LfNetwork network) {
         List<LfBranch> controllerBranches = new ArrayList<>(1);
         List<LfBranch> disabledBranches = new ArrayList<>(1);
         for (LfBranch branch : network.getBranches()) {
-            if (branch.isPhaseController() && branch.isPhaseControlEnabled()) {
+            if (!branch.isDisabled() && branch.isPhaseController() && branch.isPhaseControlEnabled()) {
```

Comment:
Your check is not equivalent to mine. I see that we have two cases:
- If the controller branch is disabled, the connectivity analysis is not necessary.
- If the controlled branch is disabled, the connectivity analysis is not necessary.
It is not exactly what we are doing, no?

Reply:
If the controlled branch is in contingency, the branch.isPhaseControlEnabled() gives false?